src/main/java/bio/terra/cli/command/helperclasses/FormatFlag.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.cli.command.helperclasses;
+
+import bio.terra.cli.command.exception.SystemException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import java.io.PrintStream;
+import java.util.function.Consumer;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the --format flag and provides utility methods to printing a
+ * return value out in different formats.
+ */
+public class FormatFlag {
+
+  @CommandLine.Option(
+      names = "--format",
+      hidden = true,
+      description = "Set the format for printing command output: ${COMPLETION-CANDIDATES}")
+  private FormatOptions format;
+
+  /** This enum specifies the format options for printing the command output. */
+  private enum FormatOptions {
+    json,
+    text;
+  }
+
+  // output stream to use for writing command return value
+  private static final PrintStream OUT = System.out;
+
+  /**
+   * This method calls the {@link #printJson} method if the --format flag is set to JSON. Otherwise,
+   * it calls the {@link #printText} method of the return value object.
+   *
+   * @param returnValue command return value
```

Comment:
not really this PR: fwiw, we're aligning on not requiring param and return tags and suppressing the javadoc warnings.

Reply:
ok sounds good. incidentally, the `javadoc` Gradle task is not passing. I added a low-priority ticket to fix this. [PF-604](https://broadworkbench.atlassian.net/browse/PF-604)